getting-started/voice/respond-to-incoming-call/server/src/main/resources/application.yaml

```
@@ -0,0 +1,15 @@
+# springboot related config file
+
+logging:
+  level:
+    com: INFO
+
+server:
+  port: 8090
+
+credentials:
+  # Application related credentials, used by:
+  # - Verification
+  # - Voice
+  application-api-key:
```

For an easier understanding from the SDK users, what do think about the proposed change?
```suggestion
  application-api-key: YOUR-APPLICATION_KEY
```

---

It could have "bad" side effect on runtime to end user.
If a value is set here, then validation against a missing credential value won't be fired on runtime.
What does this mean ?
- User will be able to run and start app (there is credentials values) so seems all is OK 
- but the webhooks validation (webhooks.validateAuthenticatedRequest) will systematically return a UNAUTHORIZED (HTTP 401) error because these values are wrong

And without any message to help end user because setup seems to be ok

-> bad onboarding/DX

So not providing a "default" value (`YOUR-APPLICATION_KEY`) will enable to not start app with an error about missing credentials.
Because of this getting started sources are linked to the getting started online doc, the online doc is having to highlight how to setup the App for execution